### DISCUSSION TITLE
Have you thought about the Difinity Internet Computer for the Stonk Terminal?

### DISCUSSION THREAD

**Discussion by crappytuna** [OpenBB-finance/OpenBB#699] [Ideas]
Seems like this project would be great on the ICP.

Check out https://dfinity.org/

Also https://www.dfinityexplorer.org

**Comment by DidierRLopes** [comment]
> Seems like this project would be great on the ICP.
> 
> 
> 
> Check out https://dfinity.org/
> 
> 
> 
> Also https://www.dfinityexplorer.org

Hey,

This is now definitely in the back of our minds.

Thanks for sharing!
